src/reformat.c

```diff
@@ -563,6 +572,19 @@ static avifResult avifImageYUVAnyToRGBAnySlow(const avifImage * image,
     const uint16_t yuvMaxChannel = (uint16_t)state->yuvMaxChannel;
     const float rgbMaxChannelF = state->rgbMaxChannelF;
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+    int maxValue;
+    if (state->mode == AVIF_REFORMAT_MODE_YCGCO_RE || state->mode == AVIF_REFORMAT_MODE_YCGCO_RO) {
+        const int bitOffset = (state->mode == AVIF_REFORMAT_MODE_YCGCO_RE) ? 2 : 1;
+        maxValue = (1 << (state->yuvDepth - bitOffset)) - 1;
```

Comment:
Nit: The variable `maxValue` is now used in a much larger scope, so its name should indicate whether it's YUV or RGB.

However, I found that we can use `state->rgbMaxChannel` in the YCgCo-R code at lines 765-767, so here we can also just do a direct comparison of bit depths as I suggested in line 210.

Reply:
Simpler indeed. So many members ... Done.